### PR TITLE
Course main page

### PR DESCRIPTION
**Pull request by ertugrul-aydin**

### CODE REVIEW

File: apps/foundation/pages/platforms/courses/index.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            >
+              Yonetmelik Kursu
+            </AcademyCard>
+            <AcademyCard
+              image="https://api.wsvvrijheid.nl/uploads/large_yol_amsterdam_activity_1f5c4f8f70.jpeg"
+              href="#"
+              description="Lorem, ipsum dolor sit amet consectetur adipisicing elit. Odio omnis libero reiciendis fugiat facere ut, necessitatibus possimus beatae et dolorem illum id neque quaerat facilis sed sit error cupiditate voluptate! Lorem"
+            >
+              Yonetmelik Kursu
+            </AcademyCard>
```
Comment (7alip) [inline apps/foundation/pages/platforms/courses/index.tsx:74]: This part should be dynamic.

Use 

```ts
const { data } = searchModel<Course>(
  url: "api/courses"
)
```

File: apps/foundation/pages/platforms/courses/index.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            <AcademyCard
+              image="https://api.wsvvrijheid.nl/uploads/large_yol_amsterdam_activity_1f5c4f8f70.jpeg"
+              href="#"
+              description="Lorem, ipsum dolor sit amet consectetur adipisicing elit. Odio omnis libero reiciendis fugiat facere ut, necessitatibus possimus beatae et dolorem illum id neque quaerat facilis sed sit error cupiditate voluptate! Lorem"
+            >
+              Yonetmelik Kursu
```
Comment (7alip) [inline apps/foundation/pages/platforms/courses/index.tsx:66]: Once you get data from the backend keep in mind that course field has translated titles (title_en, title_tr, title_nl) and descriptions.

```suggestion
              {course[`title_${locale}`]}
```

**Review by 7alip — CHANGES_REQUESTED** [review]
Thank you. I noticed that the branch had some overrides with old dev version. Maybe you used `git rebase` somewhere.

Can you pull this branch before continue working?

File: apps/foundation/pages/platforms/courses/index.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      <Container>
+        <Stack>
+          <Box>
+            <Heading pt={8} pb={5} as="h1" fontSize="4xl" textAlign="center">
+              {pagetitle}
+            </Heading>
+          </Box>
+          <Box>
+            <Text>
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Aliquam,
```
[the remark this answers is not in the document]
Comment (MustafaTRKYLMZ) [reply apps/foundation/pages/platforms/courses/index.tsx:29]: I don-t understand. Actually, I added content in to strapi.

[the remark this answers is not in the document]
Comment (7alip) [reply apps/foundation/pages/platforms/courses/index.tsx:29]: This is content about courses generally, not the content of an individual course. Where did you add this general courses detail?

[the remark this answers is not in the document]
Comment (ertugrul-aydin) [reply apps/foundation/pages/platforms/courses/index.tsx:29]: I didnt notivce strapi was ready. now i am making it dynamic.

[the remark this answers is not in the document]
Comment (MustafaTRKYLMZ) [reply apps/foundation/pages/platforms/courses/index.tsx:29]: Course static content 

https://github.com/wsvvrijheid/monorepo/wiki/Coureses-Page

**Pull request merged**